js/router.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+import Utils from "./services/Utils.js";
+import Error404 from "./views/pages/Error404.js";
+
+export default class Router {
+  constructor(routes) {
+    this.routes = routes;
+  }
+
+  load() {
+    document.getElementById("root").innerHTML = "";
```

Comment:
```suggestion
    const root = document.getElementById("root");
    while (root.hasChildNodes()) {
        root.removeChild(root.lastChild);
    }
```
C'est plus clean qu'un `innerHTML`